integration_test/testutil_test.go

```diff
@@ -276,6 +295,26 @@ func (it *ITest) initializeIndex() {
 	if err := cmd.Run(); err != nil {
 		it.t.Fatalf("cannot restore index from cache: %s", err)
 	}
+
+	pluginList := indexDir + "/plugins/"
+
+	getVersion := func(dir, plugin string) string {
```

Comment:
Determining the version is quite klunky. If you want to assert the version, why not just use plugin `foo` from `sigs.k8s.io/krew/integration_test/testdata`. This has a known, fixed version string.

Reply:
Moving to a new PR